src/applications/vaos/utils/selectors.js

```diff
@@ -402,3 +403,22 @@ export const isWelcomeModalDismissed = state =>
 
 export const selectSystemIds = state =>
   selectPatientFacilities(state)?.map(f => f.facilityId) || null;
+
+export function selectExpressCare(state) {
+  return {
+    expressCareRequests: state.appointments.future?.filter(
+      appt => appt.vaos.isExpressCare,
+    ),
+    status: state.appointments.futureStatus,
+  };
+}
+
+export function selectUpcoming(state) {
```

Comment:
Might be worth using `reselect` to cut down on the number of times this will be run, now that it will get recalculated on certain re-renders.